bin/commands/migrate.js

```diff
@@ -0,0 +1,92 @@
+// Node.js core.
+const { join } = require("path");
+
+// Enquirer engine.
+const { prompt } = require("enquirer");
+
+// Migration Helpers
+const { v4 } = require("../../lib");
+const { migratePlugin, migrateApiFolder, migrateDependencies } =
+  v4.migrationHelpers;
+
+// Prompt's configuration
+const promptOptions = [
+  {
+    type: "select",
+    name: "type",
+    message: "What do you want to migrate?",
+    choices: [
+      { name: "Project", value: "project" },
+      { name: "Only Dependencies", value: "dependencies" },
+      { name: "Plugin", value: "plugin" },
+    ],
+    result() {
+      return this.focused.value;
+    },
+  },
+  {
+    type: "input",
+    name: "pathToStrapiApp",
+    message: "Enter the path to your Strapi application",
+    initial: "./",
+  },
+];
+
+// Plugin migration prompt's configuration
+const pluginMigrationPrompt = () => {
+  return [
+    {
+      type: "input",
+      name: "name",
+      message: "Provide the name of the plugin you want to migrate",
```

Comment:
If we make the change above I suggested we will already have the path and name for the v3 plugin, so then maybe here we could change this prompt to ask where the user would like create the v4 plugin.  `Where would you like create your v4 plugin?`

Reply:
Should be fixed in latest commit 🙌🏻